src/components/MoviesList.js

```diff
@@ -0,0 +1,55 @@
+import React, {Component} from 'react';
+import MovieDetailed from './MovieDetailed'; 
+
+class MoviesList extends Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            singleMovieData:"",
+            isDetailedView: false
+        }
+    }
+    componentDidMount(){
+        this.props.sortTitles();        
+    }   
+
+    detailedMovieAPI =(id) =>{
+        fetch(`https://reactjs-cdp.herokuapp.com/movies/${id}`)
```

Comment:
Please create service api function for that and put it into services folder

Reply:
Created + Updated